app/llm_wrapper.py

```
@@ -52,6 +53,18 @@ async def ask_beam(messages: List[Dict[str, str]], endpoint: str) -> str:
             else:
                 raise ValueError("Could not get successful response from LLM")
 
+async def prewarm_beam(endpoint: str):
+    async with aiohttp.ClientSession() as session:
+        async with session.post(
+            endpoint + "warmup/",
+            headers={
+                "Content-Type": "application/json",
+                "Authorization": f"Bearer {beam_key}",
+            }
+        ) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)